### DISCUSSION TITLE
Next cache tags like fetch api for react cache function

### DISCUSSION THREAD

**Discussion by RaffoNin** [vercel/next.js#54310] [App Router] [answered]
You can add tags to Fetchs with the Fetch API according to https://nextjs.org/docs/app/building-your-application/caching#fetch-optionsnexttag-and-revalidatetag. 

Im trying to add the same functionality and revalidate data with revalidatetag as I use the same fetches in different pages. Im currently fetching data through Prisma and Im using the react cache function https://nextjs.org/docs/app/building-your-application/caching#react-cache-function. 

Is there a way to add tags to react cache function?

**Comment by viniciusbitt** [comment, marked as answer]
A little further down from what you read in the documentation there is a function called `unstable_cache` which you can use with tags:

https://nextjs.org/docs/app/building-your-application/caching#react-cache-function

if you don't care about it being **unstable**.. if you do, there is no other function that has tags :(.

**Reply by RaffoNin** [reply]
ohh I didn't get to that part. Unfortunately, it causes an error when I run it. Ill have to wait for it to be stable.

**Comment by piotrkulpinski** [comment]
Anyone know if this was implemented into the framework at some point? Can't find the info about it in the docs anymore.

**Reply by ddaoxuan** [reply]
If this would be implemented we'd probably have heard of it.

@piotrkulpinski 
https://nextjs.org/docs/app/api-reference/functions/unstable_cache
